src/api.h

```diff
@@ -8,6 +8,7 @@
 #include "device.h"
 
 #include <type_traits>
+#include <iostream>
```

Comment:
Why do you needed it ? Seem to me a debug leftover

Reply:
It fixes Linux compilation issue in [line 66](https://github.com/icarpis/librealsense/blob/bdbb42e66256a8920d9cfbae666c9019e4f36f13/src/api.h#L61) when `BUILD_EASYLOGGINGPP=false`.